datahub-web-react/src/app/lineage/utils/columnLineageUtils.ts

```diff
@@ -63,10 +63,9 @@ export function convertInputFieldsToSchemaFields(inputFields?: InputFields) {
     return inputFields?.fields?.map((field) => field?.schemaField) as SchemaField[] | undefined;
 }
 
-export function populateColumnsByUrn(
+export function getPopulatedColumnsByUrn(
```

Comment:
Wish this function had a comment or examples. It's getting complex!

Reply:
yes for sure! i'll add a doc string here - we also have some tests on it which is nice.

Reply:
tyty